SpatialGDK/Source/SpatialGDK/Private/Utils/EntityFactory.cpp

```diff
@@ -394,6 +395,17 @@ TArray<FWorkerComponentData> EntityFactory::CreateEntityComponents(USpatialActor
 		ComponentWriteAcl.Add(SubobjectInfo.SchemaComponents[SCHEMA_Handover], AuthoritativeWorkerRequirementSet);
 	}
 
+	// Add ComponentPresence to the entity.
+	if (SpatialSettings->bEnableUnrealLoadBalancer)
+	{
+		ComponentWriteAcl.Add(ComponentPresence::ComponentId, AuthoritativeWorkerRequirementSet);
+
+		// Populate the ComponentPresence component with the list of initially assigned component IDs.
+		TArray<Worker_ComponentId> ComponentPresenceList;
+		ComponentWriteAcl.GenerateKeyArray(ComponentPresenceList);
```

Comment:
Is this assuming that every component in the ACL is present on the entity at the point of creation? It sounds sane at the moment, is it something we always expect to be true?

Reply:
This also would add the entity ACL component, which AFAICT is functionally correct looking at the rest of the PR, but the semantics of the presence component seems to be "everything the simulating worker needs to simulate the actor" which doesn't include the ACL as that might be elsewhere